dotCMS/dependencies.gradle

```diff
@@ -403,4 +403,6 @@ dependencies {
     compile 'com.graphql-java:graphql-java-extended-scalars:1.0'
 
     compile 'net.jodah:failsafe:1.1.1'
+    compile 'com.rainerhahnekamp:sneakythrow:1.1.0'
```

Comment:
do we want this in here? @wezell

Reply:
The code that used it did not make it into the branch.  But it was done in a separate commit - It's a great lib and I'd vote to keep it in.  We will use it for sure.

https://github.com/rainerhahnekamp/sneakythrow#usage